chance/p5js/chance_symetry/index.md

```diff
@@ -0,0 +1,12 @@
+layout: p5js
+author: Edmundo Mejía Galindo
+author-url: https://github.com/edgalindo/gestalten-in-code/tree/master/chance/p5js/chance_symetry
```

Comment:
This points to something that is not a author url. A author url should be something like your homepage or a your github account page